yasa/hypno.py

```diff
@@ -328,6 +328,16 @@ def scorer(self):
 
     # CLASS METHODS BELOW
 
+    def copy(self):
+        """Return a new copy of the current Hypnogram."""
+        return type(self)(
+            values=self.hypno.to_numpy(),
```

Comment:
Just to be safe, should it be `.to_numpy(copy=True)`?

Reply:
Yes, good idea. Or just `.tolist()` if that also makes a copy.

Reply:
Actually I don't think this is needed since the conversion to a categorical will create a copy anyway